llvm/include/llvm/MC/CAS/MCCASObjectV1.h

```diff
@@ -443,6 +455,8 @@ class MCCASBuilder {
 
   SmallVector<MachO::any_relocation_info> AtomRelocs;
   SmallVector<MachO::any_relocation_info> SectionRelocs;
+  SmallVector<MachObjectWriter::AddendsSizeAndOffset> AtomAddends;
+  SmallVector<MachObjectWriter::AddendsSizeAndOffset> SectionAddends;
```

Comment:
The reason I don't store the reloc info with the addends is because of the `RelocLocation == CompileUnit` option, if I store the Addends in a seperate cas block, I do not know how I would reapply them, there is no way to know which Addend belongs to what section. That is why they are separate and should be separate